pkg/validation/eval.go

```diff
@@ -60,6 +62,12 @@ type Result struct {
 	failed bool
 }
 
+//Warn logs warnings into pod's log.
+//Warnings are not included in result response.
+func (r *Result) Warn(message string, e error) {
+	log.Log.V(4).Warningf(fmt.Sprintf("%s: %s", message, e.Error()))
```

Comment:
I think these warning should be enabled by default at the basic log level. Thus:
  
  log.Log.Warningf(fmt.Sprintf("%s: %s", message, e.Error()))